protocol/wlr-screencopy-unstable-v1.xml

```diff
@@ -102,6 +102,12 @@
 
         The client should then create a buffer with the provided attributes, and
         send a "copy" request.
+
+        When the buffer is a dmabuf, the stride parameter may be ignored.
```

Comment:
Should we also specify the modifier to the client?

Reply:
Are there any obvious benefits from doing so? What are the drawbacks of not allowing the client to choose its modifiers?

Reply:
I was worried about some renderers not being able to perform tiling conversions. However it seems like `vkCmdCopyImage` doesn't care about tiling, so it should be fine.